Recrutify/Recrutify.Host/CustomExceptionMiddleware/ExceptionMiddleware.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Recrutify.Host.CustomExceptionMiddleware
```

Comment:
move to Infrastructure folder / namespace

Reply:
I dont understand

Reply:
ExceptionMiddleware should be located in Infrastructure folder of Recrutidy.Host project. Now it is located in CustomExceptionMiddleware folder - it's incorrect. Namespace should be changed too from Recrutify.Host.CustomExceptionMiddleware to Recrutify.Host.Infrastructure

Reply:
ok